server/master/src/test/java/org/apache/accumulo/master/replication/SequentialWorkAssignerTest.java

```diff
@@ -66,30 +66,29 @@ public void basicZooKeeperCleanup() {
 
     queuedWork.put("cluster1", cluster1Work);
 
-    assigner.setClient(client);
-    assigner.setZooCache(zooCache);
-    assigner.setWorkQueue(workQueue);
+    assigner.zooCache = zooCache;
+    assigner.workQueue = workQueue;
```

Comment:
These should be replaced with a single call to `assigner.assignWork()`

Reply:
Gotcha that would create the zooCache and workQueue but we would still need to assign them to a "Mock" version of it right?

Reply:
You would have to do it a bit differently and setup the EasyMock objects first in the init() method, similar to how we do it in other tests (see [InMemoryMapTest](https://github.com/milleruntime/accumulo/blob/c0f9e322111c8f6bcbfcaf1c67160756153b63a5/server/tserver/src/test/java/org/apache/accumulo/tserver/InMemoryMapTest.java#L112)).  Then call replay at the end of the init method.  You will have to mock each object first calling _EasyMock.expect(..)_ with _andReturn(..)_ and _anyTimes()_ using the EasyMock framework for any objects that the code being called requires.  This usually takes some trial and error, running the test to see what gets called.  The EasyMock framework will throw exceptions telling you what it is missing.  

For example, looking at `DistributedWorkQueueWorkAssigner.initializeWorkQueue()`  you will need to add this to init to mock up the InstanceID:
`    EasyMock.expect(client.getInstanceID()).andReturn("testID").anyTimes();
`